### DISCUSSION TITLE
cambiare il tema (farlo diventare scuro) del registro elettronico

### DISCUSSION THREAD

**Discussion by filipguerra** [iisgiua/giuaschool#800] [Richieste di aiuto]
cambiare il tema (farlo diventare scuro) del registro elettronico, mi è stato detto che posso farlo mediante i template con i CSS ma sono ignorante in materia, c'è qualcuno disponibile ad aiutarmi?🙏

**Comment by trinko** [comment]
Anzi tutto evita di aprire nuove discussioni per lo stesso argomento. 
Ne hai aperto una, continua su quella: se nessuno ti risponde, credi che continuando fastidiosamente a ripetere la stessa cosa invogli le persone a risponderti?
Questo viene considerato SPAM.

**Comment by trinko** [comment]
Detto questo, quando ho risposto che puoi impostare gli stili autonomamente, mi riferivo a chi gestisce il server del registro elettronico: in questo caso si possono modificare le impostazioni predefinite negli stili CSS presenti nel codice di giua@school.

Per farlo bisogna conoscere gli stili CSS. Imparare il loro uso non così difficile, visti i tanti tutorial che si trovano su Internet.
